src/plugins/vis_type_timelion/server/series_functions/es/lib/create_date_agg.js

```diff
@@ -30,7 +31,7 @@ export default function createDateAgg(config, tlConfig, scriptedFields) {
 
   dateAgg.time_buckets.aggs = {};
   _.each(config.metric, function (metric) {
-    metric = metric.split(':');
+    metric = metric.split(/:(.+)/).slice(0, OPERANDS_NUMBER);
```

Comment:
Please check  the 'percentiles' aggregation. I see that in code below that we are getting third item from metric array.  

![image](https://user-images.githubusercontent.com/20072247/114417234-1694b980-9bba-11eb-99fa-f68d684b23eb.png)

Reply:
Resolved